### DISCUSSION TITLE
【灌水】六分街疑似出现失重现象

### DISCUSSION THREAD

**Discussion by Seayay** [share121/inter-knot#2089] [其他]
今天碰到一个面色不太和善的人，我刚准备开口就飞起来了，现在还在天上，大家有什么头绪吗。

![珍贵影像](https://github.com/user-attachments/assets/83ca83d0-9bfa-4a38-9ed9-36a54090361b)

**Comment by moyumaimoyu** [comment]
这边建议小心点，昨天在六分街和哥哥吹牛逼，然后就被他打飞了，我感觉他和最近的空洞袭击有关。害怕。

**Comment by good9527-boy** [comment]
老人孩子先飞

**Comment by MrDucker** [comment]
会不会是小型空洞入侵<img src="https://bbs-static.miyoushe.com/static/2023/11/16/a6976b54880a512092a19f34c1cd3931_2328184749682335286.png" alt="铃-萌混过关" width="75" >

**Comment by Hongxuli2006** [comment]
一会儿让你飞起来！

**Comment by M1nam1Yume** [comment]
飞起来！

**Comment by Obsidia-cdl** [comment]
伙计，找我们治安局吧，记得提供身体特征，样貌之类的关键信息

**Comment by shxtxdy** [comment]
十三分街已感到明显失重感😱
